src/index.ts

```diff
@@ -43,6 +43,7 @@ export * from './utils/calldata/enum';
 export * from './utils/contract';
 export * from './utils/events';
 export * from './utils/transactionReceipt';
+export * as wallet from './wallet/wallet';
```

Comment:
I know there was a discussion about this on Discord, but if we have a better idea not to have `wallet/wallet` ? I know naming is the hardest thing in development and I'm nitpicking xD
cc @tabaktoni

Reply:
Do you prefer a file called `walletAPI.ts` ?

Reply:
One possibility is to use `export * as wallet from './wallet'` in `src/wallet/index.ts` in place of `export * from './connect'` instead of removing it (could even keep the `connect` file name).

Reply:
I went back to `connect.ts`, keeping `export * as wallet`.